Dockerfile-Prerelease

```diff
@@ -1,4 +1,4 @@
-FROM alpine:3.17.0
+FROM alpine:3.19.1
```

Comment:
Purely as a heads up.  

Sometimes the toolchain can be a nightmare with Alpine versions because of the amount of dependencies that has to be maintained.  For example, one version change ended up in a rabbit hole for days trying to find the specific dependencies/toolchain.

I know now how cumbersome this is to maintain and I can see many companies tend to go for images that make more sense.  If I hear the argument about using minimal dependencies and keeping it lightweight. That's ok if your product requires it but never at a product this size or company.

I'm just saying if we hit these problems again.  It might be worth seeking out a distroless or recommended image rather burning time for very very very very little gain in our use case for using Alpine.  

Just saying if you hit these problems.  Don't end up in the Alpine dependencies rabbit hole.  It really is not nice for no real gain in our scenario.